src/allotropy/parsers/thermo_fisher_qubit_flex/thermo_fisher_qubit_flex_structure.py

```diff
@@ -96,21 +92,25 @@ def create_measurement_group(data: SeriesData) -> MeasurementGroup:
                         "unit": "ng/µL",
                     },
                 },
+                custom_info=data.get_unread(skip={"Software Version"}),
             )
         ],
     )
 
 
 def create_data(df: pd.DataFrame, file_path: str) -> Data:
     header = df_to_series_data(df.head(1))
+    software_version = header.get(str, "Software Version")
+    header.get_custom_keys("Sample Volume \\(uL\\)")
+    header.get_unread()
```

Comment:
I generally agree with this approach of calling `get_unread` to discard unread keys for the header. 

However, can you put a comment explaining this - for those less familiar
```suggestion
    # We reader header info from a row in the table, so we don't need to read all keys from this SeriesData.
    # Call get_unread() to mark all keys as read and suppress the warning for unread keys.
    header.get_unread()
```